content/cryptography/addresses/slides.md

```diff
@@ -114,18 +116,19 @@ _The first 5 words of the [BIP39 English dictionary](https://github.com/bitcoin/
 
 ## Mnemonic to Secret Key
 
-Of course, the secret key is a point on an elliptic curve, not a phrase.
-
-BIP39 applies 2,048 rounds of the SHA-512 hash function<br />to the mnemonic to derive a 64 byte key.
+The secret key is a scalar value from the scalar field of the base field which the elliptic curve is defined over. Not a phrase.
```

Comment:
This is probably too detailed for the level of math the cryptography module should talk about, and the number of words and terms will just make people confused.

 Probably better to phrase it as either "a 64-byte value" (which is a bit imprecise, but more black-boxy) or "a mathematical construct specific to the cryptography you're using" (which is broadly accurate, and doesn't necessitate readers to try to figure out what a "scalar value", "scalar field", or "base field" are.

Reply:
I am correcting the incorrect statement which was made that a secret key is a point on an elliptic curve which is just a false statement. If math is going to be talked about it should atleast be correct.

Reply:
A secret key is not a curve point

Reply:
Instead we can just not reference any elliptic curve. But to be honest we are going to introduce a ZK section some small references to curves and fields isnt that bad. There isnt even a single actual math line in here I kind of disagree that it is too much math. I am not showing the scalar field times the base field element or something.

Reply:
I agree that we should be correct everywhere we do reference the math, and should for sure correct what's in here right now. 

If you want to keep the definition as written, I'm fine with that if you add something in the notes for the lecturer to say that the definition is not important to know. My thought was that the definition written is a bit long and contains a few terms most won't know, and as people are learning most everything for the first time, it would be distracting.